experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -48,6 +52,9 @@ def file_path(self, version: NimbusFeatureVersion = None):
 
     @lru_cache  # noqa: B019
     def fml_client(self, version: Optional[NimbusFeatureVersion] = None) -> FmlClient:
+        """There is a single FmlClient for each combination of application, app version,
```

Comment:
This should be a comment instead of a doc string. Or the docstring should be reworded so the first line is a summary e.g.,

```
"""Return the fml client for the given version.

Clients are cached for each combination of application, version, and channel.
"""
```